cypress/fixtures/requests/view_specific_request_with_pending_offers.json

```diff
@@ -0,0 +1,22 @@
+{
+  "request": {
+    "id": 1,
+    "title": "Change to winter tires",
+    "description": "One of the wheels came off and I'm not a handy person. You might need some wrench or something as well. Will tip quick fixer.",
+    "reward": 100,
+    "offers": [
+      {
+        "id": 1,
+        "email": "helper@mail.com",
+        "status": "pending",
+        "message": "Hi! I'd love to help you with your wheel-chair and I have the tools needed. Although I think 140p would be better. Deal? When can I come // Benedict "
+      },
+      {
+        "id": 2,
+        "email": "betterHelper@mail.com",
+        "status": "pending",
+        "message": "Im the lejoin, Im the king in Sweden, can do anything. I dont need the tip, im so rich.I'll arrive fast in my lambo. Later, Ibrahimovich"
+      }
+    ]
+  }
+}
```

Comment:
since this is basically the same as the other fixture `view_specific_request_with_offers.json` one of these could be removed

Reply:
true that, I fix and update!